csm/conf/reset.py

```diff
@@ -32,5 +38,77 @@ async def execute(self, command):
         :param command:
         :return:
         """
-        Log.info("Not Preforming any Reset Since no Test Data is Generated by CSM.")
+        try:
+            Log.info("Loading Url into conf store.")
+            Conf.load(const.CSM_GLOBAL_INDEX, const.CSM_CONF_URL)
+            Conf.load(const.DATABASE_INDEX, const.DATABASE_CONF_URL)
+        except KvError as e:
+            Log.error(f"Configuration Loading Failed {e}")
+            raise CsmSetupError("Could Not Load Url Provided in Kv Store.")
+        self.disable_and_stop_service()
```

Comment:
We should only proceed if service is stopped successfully

Reply:
Modifications done